obsreport/obsreport_test.go

```diff
@@ -350,7 +350,7 @@ func Test_obsreport_ExportMetricsOp(t *testing.T) {
 	checkValueForSumView(t, "exporter/send_failed_metric_points", exporterTags, failedToSendPoints)
 }
 
-func Test_obsreport_ReceiveWithLongLivedCtx(t *testing.T) {
+func Test_obsreport_ReceiveWithLongLivedCtxAndNoTransport(t *testing.T) {
```

Comment:
It's a little untidy to use this test for two purposes - it seemed overkill to create a new test just to validate the transport attribute being optional given the boilerplate required, but I'm happy to refactor and add another test instead if anyone disagrees.

Reply:
Do we have the old path covered in other tests? If not, I don't think we should change it here to the `transport=""`. Can we have subtests covering both paths?

Reply:
The other case of transport not being empty was covered in the other tests above, but this was still pretty dirty. I've changed this test back to how it was and parameterised the other tests instead.